sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp

```diff
@@ -5,63 +5,108 @@
-
-  struct BreakBlobLeaseResult
-  {
-    std::string RequestId;
-    std::string ETag;
-    Azure::Core::DateTime LastModified;
-    int32_t LeaseTime = 0;
-  };
-
-  struct ChangeBlobLeaseResult
-  {
-    std::string RequestId;
-    std::string ETag;
-    Azure::Core::DateTime LastModified;
-    std::string LeaseId;
-  };
-
-  struct ReleaseBlobLeaseResult
-  {
-    std::string RequestId;
-    std::string ETag;
-    Azure::Core::DateTime LastModified;
-  };
-
-  struct RenewBlobLeaseResult
-  {
-    std::string RequestId;
-    std::string ETag;
-    Azure::Core::DateTime LastModified;
-    std::string LeaseId;
-  };
-
-}}}} // namespace Azure::Storage::Blobs::Models
+namespace Azure { namespace Storage { namespace Blobs {
+
+  class BlobClient;
+  class PageBlobClient;
+
+  namespace Models {
+
+    struct DownloadBlobToResult
+    {
+      std::string ETag;
+      Azure::Core::DateTime LastModified;
+      int64_t ContentLength = 0;
+      BlobHttpHeaders HttpHeaders;
+      Storage::Metadata Metadata;
+      Models::BlobType BlobType;
+      bool IsServerEncrypted = false;
+      Azure::Core::Nullable<std::vector<uint8_t>> EncryptionKeySha256;
+    };
+
+    using UploadBlockBlobFromResult = UploadBlockBlobResult;
+
+    struct AcquireBlobLeaseResult
+    {
+      std::string RequestId;
+      std::string ETag;
+      Azure::Core::DateTime LastModified;
+      std::string LeaseId;
+    };
+
+    struct BreakBlobLeaseResult
+    {
+      std::string RequestId;
+      std::string ETag;
+      Azure::Core::DateTime LastModified;
+      int32_t LeaseTime = 0;
+    };
+
+    struct ChangeBlobLeaseResult
+    {
+      std::string RequestId;
+      std::string ETag;
+      Azure::Core::DateTime LastModified;
+      std::string LeaseId;
+    };
+
+    struct ReleaseBlobLeaseResult
+    {
+      std::string RequestId;
+      std::string ETag;
+      Azure::Core::DateTime LastModified;
+    };
+
+    struct RenewBlobLeaseResult
+    {
+      std::string RequestId;
+      std::string ETag;
+      Azure::Core::DateTime LastModified;
+      std::string LeaseId;
+    };
+
+    struct StartCopyBlobResult : public Azure::Core::Operation<GetBlobPropertiesResult>
+    {
+    public:
+      std::string RequestId;
+      std::string ETag;
+      Azure::Core::DateTime LastModified;
+      std::string CopyId;
+      Models::CopyStatus CopyStatus;
+      Azure::Core::Nullable<std::string> VersionId;
+
+    public:
+      GetBlobPropertiesResult Value() const override { return m_pollResult; }
+
+      ~StartCopyBlobResult() override {}
+
+    private:
+      std::string GetResumeToken() const override
+      {
+        // Not supported
+        std::abort();
+      }
+
+      std::unique_ptr<Azure::Core::Http::RawResponse> PollInternal(
+          Azure::Core::Context& context) override;
+
+      Azure::Core::Response<GetBlobPropertiesResult> PollUntilDoneInternal(
+          Azure::Core::Context& context,
+          std::chrono::milliseconds period) override;
```

Comment:
```suggestion
          std::chrono::milliseconds pollInterval) override;
```

Reply:
Nah, this is an override function.  You should create issue for core sdk team.